### DISCUSSION TITLE
G5 Sensor change failed

### DISCUSSION THREAD

**Discussion by bionictom** [Navid200/xDrip#44] [Dexcom]
![Screenshot_2021-06-21-07-34-24-218_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122711828-250aeb80-d263-11eb-8d9c-d6de9ac7575f.jpg)
Hi, my G5 sensor was due to change and I did. It stopped working after. After the warm-up I was asked for calibration, which I entered, and it showed me the BG value. Since then, there's nothing.System Status says: "Got data" and there's nothing in the logs.

One strange thing is that Transmitter resistance jumped up. It was something like 800 a couple of hours ago, now its 1140.

Today in the morning I got the little popup that says.... got 1/3 readings, next reading expected in...... and it only got to 2 readings and nothing more after that.

![Screenshot_2021-06-20-21-31-15-866_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122711723-f260f300-d262-11eb-8bd8-cdb2351955d3.jpg)
![Screenshot_2021-06-20-21-31-32-781_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122711727-f2f98980-d262-11eb-8a8d-6728727b1777.jpg)
![Screenshot_2021-06-21-07-07-05-861_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122711732-f3922000-d262-11eb-9a05-128fae710323.jpg)

**Comment by Navid200** [comment]
Do I understand correctly that you are using the native mode but but have Fallback to xDrip enabled?
Would you mins showing a snapshot of the following page?
Settings -> G5/G6 Debug Settings

**Reply by bionictom** [reply]
I'm not using native mode, Fallback is greyed out but ticked.
![Screenshot_2021-06-21-13-01-04-003_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122752138-dd02bd80-d290-11eb-806b-74bcd4b0fa50.jpg)

**Comment by Navid200** [comment]
Are any of the settings different than these?
Never mind the grayed out ones.

https://github.com/Navid200/xDrip/wiki/G5-Recommended-Settings

**Reply by bionictom** [reply]
Yes, all the settings are like the recommended ones (except for the greyed out ones in my screenshot)

**Reply by Navid200** [reply]
Have you tapped on restart collector on the classic status page?  If not, do that.

Bye the way, the system status page says you have preemptive restart enabled.  You don't need that with non-native mode.  I would disable it if I were you.

**Reply by bionictom** [reply]
Yes, I did that a couple of times, no dice so far, gonna try again. I will change the setting.

**Reply by Navid200** [reply]
Do you see "Last connected" on your system status page.  It shows 3 minutes ago in the screenshot you have posted.
Does that keep rolling over back to 0 just before reaching 5 minutes?

**Reply by bionictom** [reply]
Yes, right now it says, got data 13:32 and last connected just jumped from 2 minutes to 3 minutes.

When I hit restartcollector, G5G7 status goes to:
![Screenshot_2021-06-21-13-36-44-493_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122755799-cdd23e80-d295-11eb-8969-f262edbabee5.jpg)

**Reply by Navid200** [reply]
No need to tap on restart collector if last connected keeps rolling over to 0.  Means connectivity is fine.

Can you show the calibration graph?

**Reply by bionictom** [reply]
![Screenshot_2021-06-21-13-42-35-745_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/28786121/122756372-9f089800-d296-11eb-8359-31b18d0b2b7e.jpg)
Calibration Graph is empty, says (no calibration)

**Reply by Navid200** [reply]
I suggest you tap on stop sensor and stop it.
Wait at least 5 minutes.
Tap on start sensor.  When it asks you when you inserted the sensor, enter the time for 2 hours ago.
In 15 minutes, it will ask for initial calibrations.   Use the existing pop-up form for entering the calibrations then.

**Reply by Navid200** [reply]
What version of xDrip is this?
You can see on the first line on classic status page.

**Reply by bionictom** [reply]
I updated a few days ago:
cc96742-2021.06.19
code:2106191119

I did the stop and am now restarting. I'm gonna report if something is working/not working

**Reply by Navid200** [reply]
Something went wrong when you entered your initial calibrations.  I am not 100% sure why.
The restart should hopefully sort it out.

**Reply by bionictom** [reply]
Thanks, it is working now.

**Reply by Navid200** [reply]
Fantastic
